### PR TITLE
@kanaabe: Adds a sticky, non-functional CTA footer

### PR DESCRIPTION
**Pull request by xtina-starr**
This current sticky footer is non-functional but has the styling that make it responsive.

screen shot 2018-02-07 at 5 57 00 pm

Signed-off-by: Yuki Nishijima <yuki@artsymail.com>

### CODE REVIEW

File: desktop/apps/the_armory_show/components/ArmoryShowFairWeekPage.js [hunk 1]
```diff
@@ -59,7 +60,40 @@ const theme = {
...
+  cursor: pointer;
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  height: 100px;
+  width: 100%;
+  background-color: #fff;
+  z-index: 999;
+  border-top: 2px solid #6e1fff;
```
Comment (kanaabe) [inline desktop/apps/the_armory_show/components/ArmoryShowFairWeekPage.js]: Could be `border-top: 2px solid ${colors. purpleRegular};`

File: desktop/apps/the_armory_show/components/ArmoryShowFairWeekPage.js [hunk 2]
```diff
@@ -59,7 +60,40 @@ const theme = {
...
+`
+
+const CtaImage = styled.img`
+  box-flex: 1;
```
Comment (kanaabe) [inline desktop/apps/the_armory_show/components/ArmoryShowFairWeekPage.js]: Never saw this attr before, but it looks like support for it is pretty low: https://developer.mozilla.org/en-US/docs/Web/CSS/box-flex

Is it similar to flex-grow?

File: desktop/apps/the_armory_show/components/ArmoryShowFairWeekPage.js [hunk 3]
```diff
@@ -130,26 +164,52 @@ export default ({ introduction, fair_coverage, event, prepare_for_fairs }) =>
...
                   </Text>
                 </a>
               </Col>
             </Row>
-          )}
+          ))}
         </Col>
       </Row>
+      <Cta>
+        <Container style={{ padding: "20px 0", display: "flex" }}>
```
Comment (kanaabe) [inline desktop/apps/the_armory_show/components/ArmoryShowFairWeekPage.js]: This styling could alternatively live within styled-components in the CTA definition --
```
const CTA = styled.div`
   ${Container} {
      padding: 20px 0;
      display: flex;
   }
`
```

**Comment by xtina-starr** [comment]
@kanaabe we refactored this PR with your suggestions.

**Pull request merged**